### DISCUSSION TITLE
what's needed to run Artisan on raspberry pi aarch64

### DISCUSSION THREAD

**Discussion by ju-emb** [artisan-roaster-scope/artisan#1667] [Q&A]
I'm completely new here, try to get familiar with Artisan.

however I'm familiar with embedded and SBC's and willing to contribute.

My question is, what makes the deb package for raspi is only 32 Bit?
What's needed to make a 64 bit deb? (new Raspi's are all aarch64)
Is there armhf bin stuff in the package?

thanks
Ju

PS: appreciate your work, nice done.

**Comment by MAKOMO** [comment]
[The recommended Raspberry Pi OS for most users](https://www.raspberrypi.com/software/operating-systems/) is still 32bit. You can run from source or build your own 64bit binary at any time.
